### PR TITLE
Add --kubeconfig support for compat with kubectl

### PR DESCRIPTION
**Pull request by smarterclayton**
@deads2k we're supporting both flags to work with upstream

### CODE REVIEW

File: pkg/cmd/cli/cli.go [hunk 1]
```diff
@@ -162,10 +162,7 @@ func NewCmdKubectl(name string, out io.Writer) *cobra.Command {
 	cmds.Aliases = []string{"kubectl"}
 	cmds.Use = name
 	cmds.Short = "Kubernetes cluster management via kubectl"
-	cmds.Long = cmds.Long + `
-
-This command exposes the exact semantics of the Kubernetes command line client with additional
-support for application lifecycles.`
+	cmds.Long = cmds.Long
```
Comment (deads2k) [inline pkg/cmd/cli/cli.go]: This looks weird.  Why the change and why the assignment?

**Comment by deads2k** [comment]
Just the question/issue with the "do nothing" assignment.  Otherwise, lgtm.

**Comment by smarterclayton** [comment]
Doh, should have nuked.

**Comment by smarterclayton** [comment]
updated [merge]
